src/elements.hpp

```diff
@@ -82,10 +81,9 @@ class G2Element {
         g2_set_infty(q);
     }
 
-    static G2Element FromBytes(const Bytes& bytes);
-    static G2Element FromBytesUnchecked(const Bytes& bytes);
+    static G2Element FromBytes(Bytes bytes);
+    static G2Element FromBytesUnchecked(Bytes bytes);
     static G2Element FromByteVector(const std::vector<uint8_t> &bytevec);
-    static G2Element FromByteVectorUnchecked(const std::vector<uint8_t> &bytevec);
```

Comment:
Shouldn't we have `FromByteVectorUnchecked` if we have the other 3, for consistency?